src/pages/dev/reference/glossary.mdx

```diff
@@ -0,0 +1,59 @@
+# Axelar Glossary
+
+The purpose of this document is to provide you with a better understanding of the terminology used in the Axelar documentation. 
+
+## Canonical
+
+A canonical token in blockchain is a standardized representation of a digital asset, such as a cryptocurrency or token, accepted as the standard across multiple blockchains in a network. This standardization ensures consistent behavior and enables interoperability between blockchains, allowing cross-chain transactions and interactions within blockchain networks that support various chains. 
+
+In the context of Axelar's Interchain Token Service, a canonical token is an interchain token built from an existing token that is available on multiple chains. The token deployed to other chains will be a [Standardized](#standardized) token, and there can only be one canonical token for every ERC-20.
+
+## Custom
```

Comment:
Ditto about needing to specify this as token. "Custom" doesn't mean this everywhere we use that word.

Reply:
Updated